content/en/docs/concepts/cluster-administration/nodes.md

```diff
@@ -0,0 +1,276 @@
+---
+reviewers:
+- reylejano
+title: Nodes
```

Comment:
Try this:
```suggestion
title: Node shutdowns
```

Reply:
(You could change the filename to match)

Reply:
Thanks for reviewing. 
Do you mean change `nodes.md` to `nodes_shutdown.md`? What if somebody wants to add more node-related content under the same section, do they have to create a new `nodes_foo.md` file?

Reply:
`node-shutdown.md` with a `-`, but yes. Having another page in the section about nodes is very likely; we already have one, and there are [plans](https://github.com/kubernetes/website/pull/45802) to rename it.

Reply:
Done. Thanks